src/git.sh

```
@@ -1,42 +1,393 @@
+  
+  last_tag=$(git for-each-ref --sort=-taggerdate --format '%(refname:short)' refs/tags | head -n 1)
+  if [ -z "$last_tag" ]; then
+    last_tag=$(git rev-list --max-parents=0 HEAD)
+  fi
+  if [ "$store" = "true" ]; then
+    echo "last_tag=${last_tag}"
+    echo "last_tag=${last_tag}" >> $GITHUB_OUTPUT
+  else
+    echo "${last_tag}"
+  fi
 }
```

Shellcheck (0.10.0)

[warning] git_get_last_created_tag references arguments, but none are ever passed.

(SC2120)